unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FeatureController/FeatureTemplate/Feature.cs

```diff
@@ -0,0 +1,8 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Feature
```

Comment:
The same as before... it would be fine to add some kind of description about what this class represents.